src/Glpi/Form/QuestionType/AbstractQuestionTypeShortAnswer.php

```diff
@@ -56,6 +56,7 @@ public function getFormEditorJsOptions(): string
     {
         return <<<JS
             {
+                "allowAnonymous": true,
```

Comment:
I think it should be possible to avoid adding new javascript logic here.
We have all the information we need on the backend and the value doesn't change once set.